rules/windows/process_creation/proc_creation_win_browsers_chromium_headless_exec.yml

```
@@ -26,7 +27,14 @@ detection:
             - '\opera.exe'
             - '\vivaldi.exe'
         CommandLine|contains: '--headless'
-    condition: selection
+    filter_main_cli:
+        CommandLine|contains:
+            - '--headless --disable-gpu --disable-extensions --disable-plugins --mute-audio --no-first-run --incognito --aggressive-cache-discard --dump-dom'
+            - '--type=gpu-process --headless --use-angle=swiftshader-webgl --headless --gpu-preferences='
+            - '--type=utility --utility-sub-type=network.mojom.NetworkService --lang=en-US --service-sandbox-type=none --use-angle=swiftshader-webgl --use-gl=angle --mute-audio --headless'
+            - '--type=renderer --headless --first-renderer-process --allow-pre-commit-input --disable-databases --disable-gpu-compositing'
+            - '--type=renderer --headless --allow-pre-commit-input --disable-databases --disable-gpu-compositing'
+    condition: selection and not 1 of filter_main_*
```

```suggestion
    condition: selection
```
when switching to low we definitely don't need to filter "known" CLI argument patterns